### ISSUE TITLE
Installing clangd in vscode breaks simulation and debugging

### ISSUE THREAD

**Issue by r4stered** [theVerySharpFlat/gradleRIO_CompilationDatabase#6]
After installing the clangd extension in vscode, it asks to disable the C++ intellisense engine for it to work correctly, but after reloading the windows and trying to run "simulate robot code" or anything related to C++, vscode comes up with an error saying "no deployers found"

![image](https://github.com/theVerySharpFlat/gradleRIO_CompilationDatabase/assets/6174102/5dc30298-0f1a-42fe-98e8-49f8199c6a7a)
![image](https://github.com/theVerySharpFlat/gradleRIO_CompilationDatabase/assets/6174102/85664e0c-7260-468e-9fc9-1468158e315f)

Im not sure how to fix this, but it seems like it has something to do with clangd extension disabling some feature of the c++ extension

**Comment by theVerySharpFlat** [comment]
Well, as a [neovim](https://xkcd.com/378) user, I generally use the command line, so I use `./gradlew simulateNative` to run the simulation program. Perhaps there's an issue with `gradle-cpp-vscode`. I'll take a look.

**Comment by theVerySharpFlat** [comment]
> Im not sure how to fix this, but it seems like it has something to do with clangd extension disabling some feature of the c++ extension

Yeah, somehow the CPP intellisense is conflicting with clangd. All you have to do (I'm 99% sure) is modify your `$projectRoot/.wpilib/wpilib_preferences.json` and set the "enableCppIntellisense" flag to false.
